### ISSUE TITLE
enable_udp stopped working

### ISSUE THREAD

**Issue by clementhk** [shadowsocks/shadowsocks-rust#147]
enable_udp still works with shadowsocks-v1.7.0-alpha.19 but not shadowsocks-v1.7.0-alpha.20.

**Comment by zonyitoo** [comment]
Yes. Use `"mode": "tcp_and_udp"` instead. I will add this breaking change into release note.

**Comment by zonyitoo** [comment]
Already updated in https://github.com/shadowsocks/shadowsocks-rust/releases/tag/v1.7.0-alpha.20 . Thanks for reporting.
